students/Tyler Ray/session1/break_me.py

```diff
@@ -0,0 +1,44 @@
+"""
+breakme
+"""
+
+def name_error():
+    print("the name error function")
+    printd("something") #spelled print wrong--aka type or reference something that doesn't exist
+
+def type_error():
+    print("Please input an integer to solve the following equation: X + 1")
+    vx = input() # Does not require an int input therefore the following won't work:
+    print(vx + 1) # Can't add an integer and a string
+
+def syntax_error(): # As just one example of a syntax error: If i removed the ':' from syntax_error() then I'd get a fatal syntax error
```

Comment:
yes, this was the idea -- you can't recover from a syntax error. The file won't compile, and none of the code in the file will run.

The idea was to get used to that :-)